README.md

```diff
@@ -114,7 +128,7 @@ cd js/src
 ./tests/jstests.py ../../obj-tf-release/dist/bin/js taint/
 ```
 
-## Citations
+## Cite us!
 
 If you would like to use project Foxhound in your own research, please cite our EuroS&P paper where we describe the browser:
```

Comment:
I suggest we also add a comment to open an issue once the paper has been accepted so we can update the wiki.

Reply:
Good idea, will do tomorrow

Reply:
Completely forgot about this. Last push adds this :)